changelogs/unreleased/gh-10148-fix-small-crash-low-alloc-factor.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/box
```

Comment:
Please bump the small library in a separate commit.

Reply:
Done.